flink-contrib/flink-storm/src/main/java/org/apache/flink/storm/util/SplitStreamType.java

```diff
@@ -20,10 +20,11 @@
 import org.apache.flink.streaming.api.datastream.DataStream;
 
 /**
- * Used by {@link org.apache.flink.storm.wrappers.AbstractStormCollector AbstractStormCollector} to wrap
+ * Used by org.apache.flink.storm.wrappers.AbstractStormCollector to wrap
```

Comment:
Javadoc couldn't link that class. I'm suspecting because AbstractStormCollector has package access and this link has been created from outside of that package. The least I could do was to leave raw path to that class.